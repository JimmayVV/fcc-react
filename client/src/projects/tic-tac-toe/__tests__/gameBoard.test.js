import React from "react";
import { render } from "react-testing-library";
import Cell from "../cell";
import GameBoard from "../gameBoard";

// Redux
import store from "../stores/gameStore";
import { Provider } from "react-redux";

function renderWithProviders(ui, options) {
  return render(<Provider store={store}>{ui}</Provider>, options);
}

it("should have 9 cells", () => {
  const { container } = renderWithProviders(<GameBoard />);
  expect(container.firstChild).toMatchSnapshot();
});
