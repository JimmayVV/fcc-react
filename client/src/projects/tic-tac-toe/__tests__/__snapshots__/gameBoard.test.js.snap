// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should have 9 cells 1`] = `
<div
  class="ticTacToe-board"
>
  <div
    class="ticTacToe-cell"
    role="button"
    tabindex="-1"
  >
    <div
      class="ticTacToe-cell__content"
    />
  </div>
  <div
    class="ticTacToe-cell"
    role="button"
    tabindex="-1"
  >
    <div
      class="ticTacToe-cell__content"
    />
  </div>
  <div
    class="ticTacToe-cell"
    role="button"
    tabindex="-1"
  >
    <div
      class="ticTacToe-cell__content"
    />
  </div>
  <div
    class="ticTacToe-cell"
    role="button"
    tabindex="-1"
  >
    <div
      class="ticTacToe-cell__content"
    />
  </div>
  <div
    class="ticTacToe-cell"
    role="button"
    tabindex="-1"
  >
    <div
      class="ticTacToe-cell__content"
    />
  </div>
  <div
    class="ticTacToe-cell"
    role="button"
    tabindex="-1"
  >
    <div
      class="ticTacToe-cell__content"
    />
  </div>
  <div
    class="ticTacToe-cell"
    role="button"
    tabindex="-1"
  >
    <div
      class="ticTacToe-cell__content"
    />
  </div>
  <div
    class="ticTacToe-cell"
    role="button"
    tabindex="-1"
  >
    <div
      class="ticTacToe-cell__content"
    />
  </div>
  <div
    class="ticTacToe-cell"
    role="button"
    tabindex="-1"
  >
    <div
      class="ticTacToe-cell__content"
    />
  </div>
</div>
`;
